src/components/FlowRunsFilterGroup.vue

```diff
@@ -15,8 +15,8 @@
       <p-label label="Deployments">
         <DeploymentCombobox v-model:selected="deployments" empty-message="All deployments" />
       </p-label>
-      <p-label label="Work Queues">
-        <WorkQueueCombobox v-model:selected="workQueues" empty-message="All work queues" />
+      <p-label label="Work Pools">
+        <WorkPoolCombobox v-model:selected="workPoolName" empty-message="All pools" />
```

Comment:
Yes, we'll be only filtering by work pool on this page since there's no mention of work queues there, and it isn't apparent to filter by something we can't see. Here's old screenshot to show why that change is needed, since we use workPools in flowRunListItems now

<img width="717" alt="Screenshot 2023-02-08 at 10 09 47 AM" src="https://user-images.githubusercontent.com/40467112/217620788-60e99074-f500-4e9a-ac5c-93030bd836f2.png">